proxy/src/test/resources/com/wavefront/agent/preprocessor/preprocessor_rules.yaml

```diff
@@ -395,3 +395,15 @@
     action        : spanAddTag
     key           : multiPortSpanTagKey
     value         : multiSpanTagVal
+
+  ## obfuscate Tests
+  - rule    : example-obfuscation
+    action  : obfuscate
+    key     : 1234567890123456
+    scope   : metricName
+
+  - rule    : example-obfuscation-2
+    action  : obfuscate
+    key     : 1234567890123456
+    scope   : hostName
```

Comment:
It seems like sourceName is the standard keyword here for matching hosts.  and you would need a match parameter
Examples
  # block all spans with sourceName that starts with qa-service
###############################################################
  - rule    : example-span-block-qa-services
    action  : spanBlock
    scope   : sourceName
    match   : "qa-service.*"